science-platform/values-minikube.yaml

```diff
@@ -36,6 +36,8 @@ nublado2:
   enabled: true
 obstap:
   enabled: true
+plot_navigator:
+  enabled: false
```

Comment:
I'm not sure if I'm supposed to turn this on for minikube? I don't know if it's for CI or for demo purposes.

Reply:
it probably won't work since I don't think the butler parts work there right now, but it was useful just for testing and such.